npm/javy-cli/index.js

```diff
@@ -73,40 +71,27 @@ async function downloadBinary() {
 		});
 	});
 
-	await fs.promises.chmod(binaryPath(), 0o775);
+	await fs.promises.chmod(binaryPath(version), 0o775);
 }
 
-async function binaryUrl() {
-	let version = process.env.FORCE_RELEASE;
-	// If no version is forced, use the GitHub API to grab the latest release.
-	if (!version || version?.toLowerCase() === "latest") {
-		const releaseDataResponse = await fetch(
-			`https://api.github.com/repos/${REPO}/releases?per_page=3`,
-			{
-				headers: {
-					Accept: "application/vnd.github+json",
-				},
-			}
+async function getDesiredVersionNumber() {
```

Comment:
Can we add a comment explaining how/why this approach works? I understand now, but I don't trust that future me or a future maintainer will necessarily remember.